AzureCalling/app/src/main/java/com/azure/samples/communication/calling/external/calling/CallingContext.java

```diff
@@ -56,7 +58,7 @@
     Context appContext;
 
     private final Callable<String> tokenFetcher;
-    private String groupId;
+    private String joinId;
```

Comment:
We should consider applying these renaming changes in main branch first to minimize changes later on when merging.